client/my-sites/site-settings/jetpack-credentials/credentials-setup-flow/setup-footer.jsx

```diff
@@ -13,35 +13,64 @@ import Gridicon from 'gridicons';
 import HappychatButton from 'components/happychat/button';
 import { recordTracksEvent } from 'state/analytics/actions';
 import isHappychatAvailable from 'state/happychat/selectors/is-happychat-available';
+import Button from 'components/button';
+import Popover from 'components/popover';
 
 class SetupFooter extends Component {
-	componentWillMount() {
-		this.setState( { showPopover: false } );
-	}
+	state = {
+		isPopoverVisible: false,
+		popoverContext: null,
+	};
+
+	setPopoverContext = popoverContext => {
+		if ( popoverContext ) {
+			this.setState( { popoverContext } );
+		}
+	};
+	togglePopover = () => this.setState( { isPopoverVisible: ! this.state.isPopoverVisible } );
+	hidePopover = () => this.setState( { isPopoverVisible: false } );
 
 	render() {
 		const { happychatAvailable, translate } = this.props;
+		const { isPopoverVisible, popoverContext } = this.state;
 
 		return (
 			<CompactCard className="credentials-setup-flow__footer">
-				{ happychatAvailable
-					? <HappychatButton
-						className="credentials-setup-flow__happychat-button"
+				<Button
+					ref={ this.setPopoverContext }
+					onClick={ this.togglePopover }
+					borderless
+				>
+					<Gridicon icon="help" />
+					<span className="credentials-setup-flow__help-button-text">
+						{
+							translate( "Need help finding your site's server credentials?" )
+						}
+					</span>
+				</Button>
+				<Popover
+					context={ popoverContext }
+					isVisible={ isPopoverVisible }
+					onClose={ this.hidePopover }
+					className="credentials-setup-flow__popover"
+					position="top"
+				>
+					{
+						translate( 'You can normally get your credentials from your hosting provider. ' +
```

Comment:
The verbiage here isn't great. @MichaelArestad thoughts?

Reply:
That copy was suggested by Michelle https://github.com/Automattic/wp-calypso/issues/20116#issuecomment-346473081